propagators/opentelemetry-propagator-aws-xray/src/AWSXRayPropagator.ts

```diff
@@ -28,26 +28,26 @@ import {
   INVALID_TRACEID,
   INVALID_SPANID,
   INVALID_SPAN_CONTEXT,
-} from '@opentelemetry/api';
+} from "@opentelemetry/api";
```

Comment:
Please remove these formatting changes.
They are unrelated and besides that it's likely that lint will complain.

Reply:
There we go, sorry about that. I had my prettierrc on another branch